Source/Core/VideoCommon/AbstractGfx.cpp

```diff
@@ -8,8 +8,8 @@
 #include "VideoCommon/AbstractFramebuffer.h"
 #include "VideoCommon/AbstractTexture.h"
 #include "VideoCommon/BPFunctions.h"
+#include "VideoCommon/EFBInterface.h"
```

Comment:
These changes that just have an include to them surprise me.  Is there a reason for the change vs just removing the include?  (I notice other includes are removed in this PR)

Reply:
Looks like this one can be removed. I'll re-check the other files.